### PR TITLE
Change UNIX paths

### PR DESCRIPTION
**Pull request by sagarnasit**
Issue ref: https://github.com/EasyEngine/easyengine/issues/1233

### CODE REVIEW

File: features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 define( 'EE', true );
 define( 'EE_VERSION', trim( file_get_contents( EE_ROOT . '/VERSION' ) ) );
-define( 'EE_CONF_ROOT', '/opt/easyengine' );
+define( 'EE_OPT_ROOT', '/opt/easyengine' );
```
Comment (rahul286) [inline features/bootstrap/FeatureContext.php]: `EE_OPT_ROOT` should be `EE_ROOT_DIR` or `EE_HOME_DIR`.

`opt` is part of the default. should not be part of constant itself. A user can move EE_ROOT to any dir which is not `opt`

Comment (mbtamuli) [reply features/bootstrap/FeatureContext.php]: We can go with `EE_ROOT_DIR`. @sagarnasit

Comment (sagarnasit) [reply features/bootstrap/FeatureContext.php]: Updated with the changes.

File: features/bootstrap/FeatureContext.php [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 define( 'EE', true );
 define( 'EE_VERSION', trim( file_get_contents( EE_ROOT . '/VERSION' ) ) );
-define( 'EE_CONF_ROOT', '/opt/easyengine' );
+define( 'EE_ROOT_DIR', '/opt/easyengine' );
```
Comment (kirtangajjar) [inline features/bootstrap/FeatureContext.php]: BTW `EE_CONF_ROOT` is used in almost all repos(ee core, cron, admin-tools, service, auth, site-type-wp, config etc...). So if you're renaming it, just ensure that it is done in all the repos. If you haven't raised PR in these repos raise them and then post them as comment here.

File: src/helper/Site_Letsencrypt.php [hunk 3]
```diff
@@ -53,7 +53,7 @@ class Site_Letsencrypt {
 	private $conf_dir;
 
 	function __construct() {
-		$this->conf_dir = EE_CONF_ROOT . '/acme-conf';
+		$this->conf_dir = EE_ROOT_DIR . '/acme-conf';
```
Comment (mbtamuli) [inline src/helper/Site_Letsencrypt.php]: This should be `EE_ROOT_DIR . 'services/nginx-proxy/acme-conf'`

File: src/helper/Site_Letsencrypt.php [hunk 4]
```diff
@@ -182,8 +182,8 @@ public function authorize( Array $domains, $wildcard = false ) {
 				$payload = $authorizationChallenge->toArray()['payload'];
 
 				$fs = new \Symfony\Component\Filesystem\Filesystem();
-				$fs->copy( SITE_TEMPLATE_ROOT . '/vhost.d_default_letsencrypt.mustache', EE_CONF_ROOT . '/nginx/vhost.d/default' );
-				$challange_dir = EE_CONF_ROOT . '/nginx/html/.well-known/acme-challenge';
+				$fs->copy( SITE_TEMPLATE_ROOT . '/vhost.d_default_letsencrypt.mustache', EE_ROOT_DIR . '/nginx/vhost.d/default' );
+				$challange_dir = EE_ROOT_DIR . '/nginx/html/.well-known/acme-challenge';
```
Comment (mbtamuli) [inline src/helper/Site_Letsencrypt.php]: This should be `EE_ROOT_DIR . 'services/nginx-proxy/html/.well-known/acme-challenge`

File: src/helper/Site_Letsencrypt.php [hunk 5]
```diff
@@ -548,8 +548,8 @@ public function cleanup() {
 
 		$fs = new \Symfony\Component\Filesystem\Filesystem();
 
-		$challange_dir = EE_CONF_ROOT . '/nginx/html/.well-known';
-		$challange_rule_file = EE_CONF_ROOT . '/nginx/vhost.d/default';
+		$challange_dir = EE_ROOT_DIR . '/nginx/html/.well-known';
+		$challange_rule_file = EE_ROOT_DIR . '/nginx/vhost.d/default';
```
Comment (mbtamuli) [inline src/helper/Site_Letsencrypt.php]: This should be `EE_ROOT_DIR . 'services/nginx-proxy/vhost.d`

File: src/helper/class-ee-site.php [hunk 6]
```diff
@@ -211,7 +211,7 @@ protected function delete_site( $level, $site_url, $site_fs_path, $db_data = []
 			\EE::log( "[$site_url] site root removed." );
 		}
 
-		$config_file_path = EE_CONF_ROOT . '/nginx/conf.d/' . $site_url . '-redirect.conf';
+		$config_file_path = EE_ROOT_DIR . '/nginx/conf.d/' . $site_url . '-redirect.conf';
```
Comment (mbtamuli) [inline src/helper/class-ee-site.php]: This should be `EE_ROOT_DIR . 'services/nginx-proxy/conf.d`

File: src/helper/site-utils.php [hunk 7]
```diff
@@ -123,18 +123,18 @@ function init_checks() {
...
+			$EE_ROOT_DIR = EE_ROOT_DIR;
 			if ( ! EE::docker()::docker_network_exists( GLOBAL_NETWORK ) ) {
 				if ( ! EE::docker()::create_network( GLOBAL_NETWORK ) ) {
 					EE::error( 'Unable to create network ' . GLOBAL_NETWORK );
 				}
 			}
-			if ( EE::docker()::docker_compose_up( EE_CONF_ROOT, [ 'nginx-proxy' ] ) ) {
-				$fs->dumpFile( "$EE_CONF_ROOT/nginx/conf.d/custom.conf", file_get_contents( EE_ROOT . '/templates/custom.conf.mustache' ) );
+			if ( EE::docker()::docker_compose_up( EE_ROOT_DIR . '/services', [ 'nginx-proxy' ] ) ) {
+				$fs->dumpFile( "$EE_ROOT_DIR/nginx/conf.d/custom.conf", file_get_contents( EE_ROOT . '/templates/custom.conf.mustache' ) );
```
Comment (mbtamuli) [inline src/helper/site-utils.php]: This should be `EE_ROOT_DIR . 'services/nginx-proxy/conf.d`

**Pull request merged**